prodsys/express/resources.py

```diff
@@ -88,6 +88,7 @@ class ProductionResource(Resource, core.ExpressObject):
     processes: List[Union[process.ProductionProcess, process.CapabilityProcess]]
     control_policy: resource_data.ResourceControlPolicy = resource_data.ResourceControlPolicy.FIFO
     queue_size: Optional[int] = 0
+    batch_size: Optional[int] = None
```

Comment:
I think adding the batch size as a new parameter is not really necessary. The capacity attribute should be sufficient in my sense. Adding an extra batch size might be interesting if multiple batches can be processed at once (capacity > batch size). However, I think this makes the API more complex for cases that happen not very often... I think for simple Batch resources, the batch size should be the capacity. What do you think?

Reply:
I think in the long run, we should allow use cases like the above where multiple batches can be produced at once by adding building up these logics by individuals machines that are combined to work_centers. However, this is not yet needed I think...